collector/aws/resources/rds.go

```diff
@@ -136,9 +153,50 @@ func (r *RDSManager) Detect(metrics []config.MetricConfig) (interface{}, error)
 					"formula_value":       formulaValue,
 					"name":                *instance.DBInstanceIdentifier,
 					"instance_type":       *instance.DBInstanceClass,
+					"engine":              *instance.Engine,
 					"region":              r.awsManager.GetRegion(),
 				}).Info("RDS instance detected as unutilized resource")
 
+				instancePricingFilters := r.getPricingInstanceFilterInput(instance)
```

Comment:
The price calculation really needs to be in the metrics loop? 
the price can be changed when we have a different metric?